Twig/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Sonata\FormatterBundle\Twig\Extension;
+
+use Ivory\CKEditorBundle\Templating\CKEditorHelper;
+
+/**
+ * @author Marodon Jérémy <marodon.jeremy@gmail.com>
+ */
+class CKEditorExtension extends \Twig_Extension
+{
+    /**
+     * @var CKEditorHelper
+     */
+    private $helper;
+
+    /**
+     * Creates a CKEditor extension.
```

Comment:
Yes, this is why I called it a bridge.

This bridge will keep the old method by calling the new one.